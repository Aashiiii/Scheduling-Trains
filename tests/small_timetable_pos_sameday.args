Waverton Windsor "18:00:00"
