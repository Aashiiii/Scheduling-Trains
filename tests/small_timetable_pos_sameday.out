The next train to Windsor from Waverton departs at 20:00:01
